packages/pwa-kit-mcp/README.md

```diff
@@ -1,206 +1,158 @@
-# PWA Storefront MCP Server
+# 🚀 PWA Kit MCP Server
 
-A Model Context Protocol (MCP) server that provides AI coding assistance for developing  PWA-kit-based storefront apps in an IDE.
+An MCP server that brings AI-assisted coding to **PWA Kit** storefront app development inside your IDE.
 
-Install and run this MCP server as a local MCP server or run locally via `npx` (when it's available on `npm` in feature).
+## 🔍 What is MCP?
 
-## What is MCP?
+**Model Context Protocol (MCP)** is an open protocol that enables secure, structured communication between host applications (like [Cursor IDE](https://www.cursor.so/) or other AI development tools) and external tools or data sources.
 
-The Model Context Protocol (MCP) is an open protocol that enables secure connections between host applications, such as Claude Desktop or other AI assistants, and external data sources and tools.
+It allows AI agents to query context-aware services like this server to help developers build better software, faster.
 
-## Features
+👉 **[Read more at modelcontext.org](https://modelcontext.org)**
 
-The PWA Storefront MCP Server provides these features.
 
-- `development_guidelines`: Helps developers understand and follow PWA Storefront developer guidelines and best practices.
-- `create_new_sample_component`: Helps developers create a new sample PWA Storefront component. This feature guides developers through a few simple questions and then generates code for the component based on the commerce data model used, layouts, etc.
-- `create_app_guidelines`: This tool provides all the information an agent needs to correctly scaffold a new PWA Kit app using the `@salesforce/pwa-kit-create-app` CLI. 
+## 🧰 Features
 
-  It returns:
-    - Project creation guidelines for agent behavior
-    - CLI description and available options
-    - Input schemas for presets or templates
+The PWA Kit MCP Server offers the following intelligent tools tailored to Salesforce Commerce Cloud PWA development:
 
-    The output enables agents to ask the right questions and use the CLI correctly without ever mixing unsupported options.
+* **`create_app_guidelines`**
```

Comment:
In the output, the name of the package and description appear on the same line separated by a space. Would be nice to add a colon in between.
```suggestion
* **`create_app_guidelines`**:
```